frontend/src/components/Project/Settings/ProjectIssueRelatedSettingPanel.vue

```diff
@@ -107,6 +107,19 @@
           }}
         </p>
       </div>
+      <div>
```

Comment:
> It should NOT be in 3.0.0!!!!

If it's not ready to release, we can guard it with `isDev()`.

Refer to: https://github.com/bytebase/bytebase/blob/main/frontend/src/components/Project/useProjectSidebar.ts#L147

Reply:
done